templates/donation_successful.html

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
Like I said above, I would strongly prefer a popup on the main page upon successful completion of that form as opposed to a new page, so see if you can return JSON before we go down this route.